aurorastation.dme

```diff
@@ -3725,12 +3725,12 @@
 #include "code\ZAS\Zone.dm"
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
+#include "interface\skin.dmf"
 #include "interface\fonts\fonts_datum.dm"
 #include "interface\fonts\grand_9k.dm"
 #include "interface\fonts\pixellari.dm"
 #include "interface\fonts\spess_font.dm"
 #include "interface\fonts\tiny_unicode.dm"
-#include "interface\skin.dmf"
```

Comment:
To specify, the mistake here is moving the line 3733 to line 3728; that, specifically, should be reverted.

Reply:
thank you! lly fixed it, happily